### ISSUE TITLE
 Is this project still under maintenance?

### ISSUE THREAD

**Issue by Xavier-i** [castorini/honkling#125]
Is this project still under maintenance?

**Comment by lintool** [comment]
Not actively, but something you need?

**Comment by Xavier-i** [comment]
Yes, was trying to run the electron version. But ran into some deprecated dependency issues, also not sure which node version I should use. THANKS!
